tox.ini

```diff
@@ -0,0 +1,24 @@
+[flake8]
+exclude =
+    # No need to traverse our git directory
+    .git,
+    # There's no value in checking cache directories
+    __pycache__,
+    # This contains our built documentation
+    build/*,
+    # This contains builds of flake8 that we don't want to check
+    dist/*
+max-line-length = 120
+show-source = true
+statistics = true
+
+docstring-quotes = """
+multiline-quotes = """
+
+[isort]
```

Comment:
Isn't this supposed to be pylint?

Reply:
They are. They work with `flake8-commas`, `flake8-isort` and `flake8-quotes` together with `flake8`.